std/experimental/ndslice/slice.d

```diff
@@ -805,6 +810,7 @@ Returns:
 Throws:
     $(LREF SliceException) if the array is not an n-dimensional parallelotope.
 +/
+@trusted
```

Comment:
What's this needed for? As far as I see, the most unsafe thing here is a void initialization. And that's actually accepted in `@safe`.